src/EnergyPlus/Autosizing/HeatingCapacitySizing.cc

```diff
@@ -117,28 +117,14 @@ Real64 HeatingCapacitySizer::size(EnergyPlusData &state, Real64 _originalValue,
                             CoilInTemp = (this->termUnitFinalZoneSizing(this->curTermUnitSizingNum).DesHeatCoilInTempTU * MinPriFlowFrac) +
                                          (this->termUnitFinalZoneSizing(this->curTermUnitSizingNum).ZoneTempAtHeatPeak * (1.0 - MinPriFlowFrac));
                         }
-                    } else if (this->zoneEqFanCoil) {
-                        // use fan coil flow (i.e., set by parent) or flow used during sizing?
-                        if (DesVolFlow > 0.0) {
-                            DesMassFlow = DesVolFlow * state.dataEnvrn->StdRhoAir;
-                        } else {
-                            DesMassFlow = this->finalZoneSizing(this->curZoneEqNum).DesHeatMassFlow;
-                        }
-                        CoilInTemp = this->setHeatCoilInletTempForZoneEqSizing(
-                            this->setOAFracForZoneEqSizing(state, DesMassFlow, zoneEqSizing(this->curZoneEqNum)),
-                            zoneEqSizing(this->curZoneEqNum),
-                            finalZoneSizing(this->curZoneEqNum));
-                        CoilInHumRat = this->setHeatCoilInletHumRatForZoneEqSizing(
-                            this->setOAFracForZoneEqSizing(state, DesMassFlow, zoneEqSizing(this->curZoneEqNum)),
-                            zoneEqSizing(this->curZoneEqNum),
-                            finalZoneSizing(this->curZoneEqNum));
```

Comment:
Duplicate code reported by clang-tidy. Deleting this section will allow to fall through to else where same code is executed..